tests/acceptance/features/webUISharingInternalUsers/shareAutocompletion.feature

```diff
@@ -26,27 +26,27 @@ Feature: Autocompletion of share-with names
       | users-finance |
       | other         |
 
-  @smokeTest @issue-1532
+  @smokeTest
   Scenario: autocompletion of regular existing users
     Given user "regularuser" has logged in using the webUI
     And the user has browsed to the files page
     And the user has opened the share dialog for folder "simple-folder"
     When the user types "us" in the share-with-field
-#    Then all users and groups that contain the string "us" in their name should be listed in the autocomplete list on the webUI
-    Then user "John Finn Smith" should not be listed in the autocomplete list on the webUI
-    And every item listed in the autocomplete list on the webUI should contain "us"
-    But the users own name should not be listed in the autocomplete list on the webUI
+    And the user shows all share-autocomplete results using the webUI
+    Then all users and groups that contain the string "us" in their name should be listed in the autocomplete list on the webUI
+    But only users and groups that contain the string "us" in their name or displayname should be listed in the autocomplete list on the webUI
+    And the users own name should not be listed in the autocomplete list on the webUI
 
-  @smokeTest @issue-1532
+  @smokeTest
   Scenario: autocompletion of regular existing groups
     Given user "regularuser" has logged in using the webUI
     And the user has browsed to the files page
     And the user has opened the share dialog for folder "simple-folder"
     When the user types "fi" in the share-with-field
-#    Then all users and groups that contain the string "fi" in their name should be listed in the autocomplete list on the webUI
-    Then user "User Group" should not be listed in the autocomplete list on the webUI
-    And every item listed in the autocomplete list on the webUI should contain "fi"
-    But the users own name should not be listed in the autocomplete list on the webUI
+    And the user shows all share-autocomplete results using the webUI
+    Then all users and groups that contain the string "fi" in their name should be listed in the autocomplete list on the webUI
+    But only users and groups that contain the string "fi" in their name or displayname should be listed in the autocomplete list on the webUI
```

Comment:
Does this need to be as two steps? Can't it be consolidated?

Reply:
sure, ideas for good wordings that is nice to read and tells enough?